descartes_utilities/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(descartes_utilities)
+
+find_package(catkin REQUIRED COMPONENTS
+  descartes_core
+  descartes_trajectory
+  trajectory_msgs
+)
+
+catkin_package(
+ INCLUDE_DIRS include
+ LIBRARIES descartes_utilities
+ CATKIN_DEPENDS descartes_core descartes_trajectory trajectory_msgs
+)
+
+include_directories(include)
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_library(${PROJECT_NAME}
+  src/ros_conversions.cpp
+)
+
+add_dependencies(${PROJECT_NAME} 
+  ${${PROJECT_NAME}_EXPORTED_TARGETS} 
+  ${catkin_EXPORTED_TARGETS}
+)
```

Comment:
If this package doesn't generate any messages / services of its own, then I think `${${PROJECT_NAME}_EXPORTED_TARGETS}` isn't actually needed.